IndyAV/simulation/indyav_gazebo/xacro/axles.xacro

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro" name="axles">
+  
+  <xacro:macro name="axle_physical">
+      <xacro:property name="axle_length" value="0.005"/>
+      <xacro:property name="axle_mass" value="1"/>
+      <xacro:property name="axle_radius" value="0.005"/>
+
+      <inertial>
+        <mass value="${axle_mass}"/>
+        <origin xyz="0 0 0"/>
+        <xacro:cylinder_inertia h="${axle_length}" r="${axle_radius}" mass="${axle_mass}"/>
+      </inertial>
+  </xacro:macro>
+
+
+  <!-- Front Axle Xacro -->
+  <xacro:macro name="front_axle" params="name xyz parent_link:='base_link' 
+                                                      max_deflection_left_deg:='45'
+                                                      max_deflection_right_deg:='45'">
+
+    <link name="${name}_link">
+       <xacro:axle_physical/>
+    </link>
+
+    <joint name="${parent_link}_to_${name}_link_joint" type="continuous">
```

Comment:
I think this joint should be of type `revolute` since it should not be able to spin around without limits

Reply:
If you put a limit tag on a continuous joint, it basically becomes revolute, but I will change it for clarity.